src/plugin/coreplugins/RestRouterGenerator/RestRouterTest.ejs

```diff
@@ -0,0 +1,67 @@
+var testFixture = require('../../globals'),  // TODO: May need to change this if not created from webgme-cli
```

Comment:
Would you mind moving these within the describe?
It might not be that big of a deal.. But if you have a lot of test files and a recursive look-up and only want to run some tests using `only` it will load all these modules.
(Keeping the testFixture here is fine since all tests practically use that)